### PR TITLE
Restart celery on deploy

### PR DESCRIPTION
**Pull request by mjvkeulen**
## Description
Any modifications to celery related code executed by workers requires a reload before it takes effect.

### CODE REVIEW

File: system_configuration_tools/roles/celery/tasks/restart_celery.yml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+---
+
+- name: Restart {{ celery_application_name }}_beat
+  supervisorctl:
+    name: "{{ celery_application_name }}_beat"
+    state: restarted
```
Comment (Larspolo) [inline system_configuration_tools/roles/celery/tasks/restart_celery.yml]:
```TASK [celery : Restart celery_beat] *********************************************
fatal: [test.ejournal.app]: FAILED! => changed=false 
  msg: |-
    celery_beat: ERROR (not running)
    celery_beat: ERROR (spawn error)
```
This task fails when its not running

Comment (Larspolo) [reply system_configuration_tools/roles/celery/tasks/restart_celery.yml]: image

Comment (mjvkeulen) [reply system_configuration_tools/roles/celery/tasks/restart_celery.yml]: This is desired behaviour. When **deploying** to a correctly **provisioned** server celery beat will be up and running. This is simply feedback something is wrong, which is correct.

Comment (mjvkeulen) [reply system_configuration_tools/roles/celery/tasks/restart_celery.yml]: (Can't restart something if it just crashes right)

Comment (Larspolo) [reply system_configuration_tools/roles/celery/tasks/restart_celery.yml]: Nothing else seems to fail, so what else could be going wrong? Cuz it doesnt log much else

Comment (mjvkeulen) [reply system_configuration_tools/roles/celery/tasks/restart_celery.yml]: Currently celery beat has timezones issues

**Pull request merged**